contrib/services/hierarchy_keys/ssh-tpm-agent@.service

```diff
@@ -0,0 +1,19 @@
+[Unit]
+ConditionEnvironment=!SSH_AGENT_PID
+Description=ssh-tpm-agent service
+Documentation=man:ssh-agent(1) man:ssh-add(1) man:ssh(1)
+Wants=ssh-tpm-genkeys@%i.service
+After=ssh-tpm-genkeys@%i.service
+After=network.target
+After=sshd.target
+Requires=ssh-tpm-agent@%i.socket
+
+[Service]
+ExecStart=/usr/bin/ssh-tpm-agent --key-dir /etc/ssh --hierarchy %i
+PassEnvironment=SSH_AGENT_PID
+KillMode=process
+Restart=always
+
+[Install]
+WantedBy=multi-user.target
+DefaultInstance=endorsement
```

Comment:
While I'm running it this way, the thing that's shipped (even only as contrib), should probably be
```ini
DefaultInstance=owner
```